<launch>
    <node name="rosbag_recorder" pkg="px4ctrl" type="rosbag_recorder.py" output="screen">
        <param name="drone_id" value="2" type="int"/>
    </node>
</launch>